app/views/claims/support/support_users/check.html.erb

```diff
@@ -0,0 +1,53 @@
+<% content_for(:page_title) { t(".page_title") } %>
+
+<%= content_for(:before_content) do %>
+  <%= govuk_back_link(href: new_claims_support_support_user_path) %>
```

Comment:
this back link should include the form params.

Reply:
Same as above. Now addressed.